### ISSUE TITLE
Blender 2.8 GLTF importer tries to set `weight_animation` but the attribute doesn't exist.

### ISSUE THREAD

**Issue by roxlu** [KhronosGroup/glTF-Blender-IO#364]
I'm trying to import a GLTF file that I converted from collada using the Khronos COLLADA2GLTF converter (using default options) and I get this callstack:

```
2019-03-15 11:08:39,365 CRITICAL Data are loaded, start creating Blender stuff
Traceback (most recent call last):
  File "/home/roxlu/Downloads/blender-2.80-c7cf8282a6ab-linux-glibc224-x86_64/2.80/scripts/addons/io_scene_gltf2/__init__.py", line 457, in execute
    return self.import_gltf2(context)
  File "/home/roxlu/Downloads/blender-2.80-c7cf8282a6ab-linux-glibc224-x86_64/2.80/scripts/addons/io_scene_gltf2/__init__.py", line 478, in import_gltf2
    BlenderGlTF.create(self.gltf_importer)
  File "/home/roxlu/Downloads/blender-2.80-c7cf8282a6ab-linux-glibc224-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_gltf.py", line 33, in create
    BlenderScene.create(gltf, scene_idx)
  File "/home/roxlu/Downloads/blender-2.80-c7cf8282a6ab-linux-glibc224-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_scene.py", line 76, in create
    BlenderNode.create(gltf, node_idx, None)  # None => No parent
  File "/home/roxlu/Downloads/blender-2.80-c7cf8282a6ab-linux-glibc224-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 165, in create
    BlenderNode.create(gltf, child_idx, node_idx)
  File "/home/roxlu/Downloads/blender-2.80-c7cf8282a6ab-linux-glibc224-x86_64/2.80/scripts/addons/io_scene_gltf2/blender/imp/gltf2_blender_node.py", line 61, in create
    if pynode.weight_animation is True:
AttributeError: 'Node' object has no attribute 'weight_animation'

location: <unknown location>:-1

```

**Comment by julienduroure** [comment]
Please download last version of addon (for 2.79) or last build of Blender beta 2.80 (for 2.80). 
This bug was fixed here : 3d2793fd8ff3a34d0d5da3513989580887cab065
And here for 2.80 beta : https://developer.blender.org/rBA1fe812499d5c8e2962d8d5a90b2b5837097e334e

If you still have an issue with latest version, please reopen this issue, with you glb/gltf file in attachment

**Comment by roxlu** [comment]
Ah awesome updating to the latest 2.8 fixed it.
